services/core-api/app/api/mines/incidents/resources/mine_incidents.py

```diff
@@ -246,6 +246,16 @@ def get(self, mine_guid, mine_incident_guid):
             raise NotFound("Mine Incident not found")
         return incident
 
+    @requires_role_mine_admin
+    @api.response(204, 'Successfully deleted.')
+    def delete(self, mine_guid, mine_incident_guid):
+        incident = MineIncident.find_by_mine_incident_guid(mine_incident_guid)
+        if not incident:
+            raise NotFound('Unable to fetch incident')
```

Comment:
I think something along the lines of "Mine incident not found", or whatever we typically say for 404s, would be a better message because it's less ambiguous.

Reply:
done